gapic-generator-cloud/templates/cloud/gem/authentication.erb

```diff
@@ -0,0 +1,169 @@
+<%- assert_locals gem -%>
+<%- service = gem.services.first -%>
+<%- assert_locals service -%>
+# Authentication
+
+In general, the <%= gem.name %> library uses [Service
+Account](https://cloud.google.com/iam/docs/creating-managing-service-accounts)
+credentials to connect to Google Cloud services. When running within [Google
+Cloud Platform environments](#google-cloud-platform-environments)
+the credentials will be discovered automatically. When running on other
+environments, the Service Account credentials can be specified by providing the
+path to the [JSON
+keyfile](https://cloud.google.com/iam/docs/managing-service-account-keys) for
+the account (or the JSON itself) in [environment
+variables](#environment-variables). Additionally, Cloud SDK credentials can also
+be discovered automatically, but this is only recommended during development.
+
+## Quickstart
+
+1. [Create a service account and credentials](#creating-a-service-account).
+2. Set the [environment variable](#environment-variables).
+
+```sh
+export <%= gem.env_prefix %>_CREDENTIALS=path/to/keyfile.json`
+```
+
+3. Initialize the client.
+
+```ruby
+require "<%= service.package.package_require %>
```

Comment:
There is a missing `"` from the end of this line.

Reply:
Fixed!